### PR TITLE
Bug 1677442 - Intervention for store.hp.com

### PR DESCRIPTION
**Pull request by ksy36**
r? @denschub

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "@EXTENSION_NAME@",
   "description": "Urgent post-release fixes for web compatibility.",
-  "version": "18.0.0",
+  "version": "18.0.1",
```
Comment (denschub) [inline src/manifest.json:5]: One *minor* nit: https://github.com/mozilla-extensions/webcompat-addon/wiki/Versioning-Scheme - that should probably be `18.1.0` - not that it really matters, but let's be consistent. :)

**Review by denschub — APPROVED** [review]
r+, one nit.

**Comment by ksy36** [comment]
Thanks!

**Comment by llacb47** [comment]
Maybe off-topic, but none of the 3D models load right now in any browser due to CORS errors.

Test links:

```
https://store.hp.com/ItalyStore/Merch/Product.aspx?id=8MK76EA&opt=ABZ&sel=NTB
https://store.hp.com/ItalyStore/Merch/Product.aspx?id=8MK78EA&opt=ABZ&sel=NTB
https://store.hp.com/ItalyStore/Merch/Product.aspx?id=6TW21ET&opt=ABZ&sel=NTB
```

image
image

`https://flixmedia.eu/contact/`

**Pull request merged**